pkg/api/serve_test.go

```diff
@@ -113,6 +113,11 @@ func setupHandler(t testing.TB) (http.Handler, *dependencies) {
 	// Add endpoint so that 'IsAdvancedAuth' will be in effect
 	viper.Set("auth.api.endpoint", config.DefaultListenAddress)
 
+	// Set minimal cache sizes for testing to reduce memory consumption
+	// Default production values are too large for test environments
+	viper.Set("committed.local_cache.size_bytes", 8*1024*1024)          // 8MB instead of 1GB
```

Comment:
```suggestion
	viper.Set("committed.local_cache.size_bytes", 24*1024*1024)
```
1. Remove the comment - keeping it means we need to keep this file sync'ed up with defaults.go, and we won't do that.
1. We'll anyway probably never hit this in a small test, but if we do - it will slow down the API test.